### PR TITLE
Add docs for setting up development environment on Windows

### LINKED ISSUE #7151: [Docs] Publish Windows contributor docs for Meshery Server and Meshery UI
#### Current State
Meshery Contributors that use Windows as their operating system are in need of build instructions for Meshery Server and Meshery UI. Instructions have already been written in two Google Docs. These two Google Docs can be combined and published.

Already published: [Meshery Docs with Windows](https://docs.meshery.io/project/contributing/contributing-docs#for-windows) 

#### Desired State
1. Review “[Jekyll Tutorial](https://docs.google.com/document/d/1JoPujIO_dyX3pTHvbGtQ5OBXERlSInac3ecLSkgFFSo/edit#heading=h.ipjd8jcauwx)”. document and publish any missing details into [Meshery Docs with Windows](https://docs.meshery.io/project/contributing/contributing-docs#for-windows) .

2. Combine and publish the contents of these two docs:
    1. “[Setting up your Meshery Development Environment on Windows](https://docs.google.com/document/d/1QHBb3TRhlYDTvcwodnY-KX2v73RYNclL8UPlGE5U2Y8/edit#)”
    2. “[Setting up the Meshery runtime on the local Windows Environment](https://docs.google.com/document/d/1nyFu2oD9s6SDrdo5IwDAhOzbA5-5aSzf7Xh4RZQYol0/edit)”

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by mohit-marathe**
**Notes for Reviewers**

This PR fixes #7151 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by harkiratsm** [comment]
merge conflicts

**Comment by Chadha93** [comment]
@mohit-marathe Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by leecalcote** [comment]
> netlify preview is not showing anything. What is the reason for that?

There was a bug in an earlier version of the site. I'll update here, so that it builds as expected.

**Comment by l5io** [comment]
🚀 Preview for commit 52530769c4d2e431c9e1cb9f7ed801829f10890c at: https://63fe93b4d1e7c353ff38339b--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/meshery-windows.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## On Windows Native Mode
+
+Installing Meshery on native Windows may be a little more involved than installing Meshery on a Linux platform. That is because Linux installations usually come with a lot of the prerequisites preinstalled. Moreover, one is given a hint on the Linux platform on what to do next including the command, especially on Ubuntu.
+
+On Windows, make, gcc and some of the other prerequisites do not come pre-installed, These will need to be installed manually. Also, after installing any of the items, it may be necessary to set its path. This can be done in the control panel by searching for setting the system environment variables.
+
+Please research the internet to install these components. However, here are a few suggestions:
+
+- make: [https://gnuwin32.sourceforge.net/packages/make.htm](https://gnuwin32.sourceforge.net/packages/make.htm)
```
Comment (sudhanshutech) [inline docs/pages/project/contributing/meshery-windows.md:43]: `.html` is incomplete

**Review by sudhanshutech — COMMENTED** [review]
url is incomplete

File: docs/pages/project/contributing/meshery-windows.md (hunk 1 above)
Comment (mohit-marathe) [reply docs/pages/project/contributing/meshery-windows.md:43]: There is no correction to be made here.

Comment (sudhanshutech) [reply docs/pages/project/contributing/meshery-windows.md:43]: oh okay then

**Comment by l5io** [comment]
🚀 Preview for commit 0718a2a92259a7b2831832dadc4a03bab7787e7c at: https://63ff5b4021156d0b4d1cded5--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Thank you, @mohit-marathe

**Pull request merged**